pitch.md

```diff
@@ -1,60 +1,142 @@
-# Project Title (replace me)
+# To-Do List Organizer
 
 ## Describe your project in five words
+Chronological to-do lists from categories
 
 ## Audience
 _Who are you helping, with your language? What is that person's
 experience like now? How would their experience be better if you could help 
-them?_
+them?_  
+The audience would be people who use to-do lists to organize their plans.
+Currently, either people organize a to-do list chronologically, or they do it by some bigger, more general
+categories. In ordering lists chronologically, it's easy to miss items as you have to think back and forth
+between all the categories. When putting to-do lists together by categories, on the other hand, it's hard to
+see it as a cohesive to-do list instead of multiple to-do lists that you need to juggle. In other words, it's
+easier to write but harder to use.
 
 
 ## Why a language?
-_Why is a **DSL** appropriate for your user(s)? How does it address the need?_
+_Why is a **DSL** appropriate for your user(s)? How does it address the need?_  
+It is helpful because the language can allow for a lot of flexibility in what the user wants to be able to
+write. Additionally, categorized to-do lists are already like a language. You have a large category with
+smaller tasks, each of which has a deadline. Some people create their to-do lists per day while some people 
+make them per hour. So, there is a lot of variability in the tasks and timelines, but they all follow a similar
+outline of big categories, smaller tasks, and a time scale.
 
 
 ## Why you?
-_What excites you about this idea? How did you come up with it?_
+_What excites you about this idea? How did you come up with it?_  
+I'm very excited about this project because of the usefulness of it. I know a lot of people who struggle with
+this same problem and I think that it would be very rewarding to have people use it (after, of course, the
+initial prototype struggles). 
+
+Initally, I thought of this idea because of my experiences dealing with multiple people having a bunch of
+tasks and having short, stressful periods followed by long, relaxed periods. As I have thought more 
+about this idea, though, I don't think this would be my main focus. I think that
```

Comment:
I'm not quite sure what this refers to.  I think you mean that you are not going to focus on helping multiple people pool their todo lists together to help smooth over periods of lots of work and little work. Is that correct?